experimental/go-dep/image/vendor.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+mkdir -p /go/src/project/user-app
```

Comment:
Some of this seems like setup that should only be done once in the Dockerfile-stack.
I'm thinking APPSODY_PREP should simply run `dep ensure`